README.md

```diff
@@ -1,2 +1,51 @@
 # daedalus-deploy
 Deploy tool for the Daedalus app
+
+## Installation
+
+Clone this repo and install dependencies with:
+```
+pip3 install --user -r requirements.txt
```

Comment:
we've been using hatch more recently rather than this approach.  practically that might require rescaffolding the app though. See e.g., pyorderly, privateer for examples